src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,64 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $padding-size;
+  gap: $gap-size;
+  width: calc(7 * #{$day-size} + 6 * #{$gap-size} + 2 * #{$padding-size});
```

Comment:
Calculation of width should be done using arithmetic operations in Sass, not in the `calc` CSS function, as per the Code Style Rules. Replace `calc` with arithmetic operations in Sass.